### DISCUSSION TITLE
Proposed resource deposists

### DISCUSSION THREAD

**Discussion by UberWaffe** [Keriew/augustus#885] [Ideas]
The idea is that for all resource types we introduce dedicated resource deposits as new terrain features, and per-map settings per-resource that determines if the deposits act as a speed boost (default), or as an 'exclusively mine here' deal.

By default (unless changed by the map maker for newer maps) the behaviour will be 'speed boost'.
This means that resource buildings can still be placed like they can now (ex: iron mines next to rocks, or clay pits close to water), but if placed next to their respective dedicated resource deposit (ex: iron mine next to rich iron deposit) they will produce at double speed (200%, think of how wheat works in central or desert climates).

The other map-maker option that can be set per resource is 'mine only at deposit'.
In this mode, the resource cannot be placed like it normally does. (Ex: You would not be able to place gold mines next to rocks.)
Instead, you have to place the resource building next to its respective deposit, and it operates at only the normal 100% rate. (Ex: Gold mine adjacent to a gold vein.)

The following types of deposits should exist:
Rich vein: Affects any mining (iron mines, gold mines, marble quarries)
Rich stone deposit: Affects stone quarries
Rich clay deposit: Affects clay pits
Rich marble deposit: Affects marble quarries
Dense forest: Affects timber yards

**Comment by crudelios** [comment]
This seems like a good addition to me 👍

**Comment by PaxBotanica** [comment]
For map making this would be a huge development. Rather than generic rocks the introduction of specific rocks to dictate where a player should set up mining particular resources would be amazing.
